src/style.css

```diff
@@ -1 +1,30 @@
-/* add styles here */
+.body {
```

Comment:
You're using a class selector `.body` instead of a tag selector `body`. It's recommended to use tag selectors for `html` and `body` tags. So, you should use `body` instead of `.body` here.